.github/workflows/ci.yml

```diff
@@ -61,9 +61,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@master
-        with:
-          toolchain: nightly-2023-09-13
+      - uses: dtolnay/rust-toolchain@nightly
```

Comment:
The nightly version should remain pinned for CI reproducibility.

Reply:
Pinned to... which date do you suggest?

Reply:
I don't suggest any in particular, as long as it works. Ideally, however, we'd match the Rust versions used elsewhere in CI to avoid mismatches. That'd be 1.74.0 I think (so, a nightly version around its release, or something).